packages/web-extension/src/walletManager/walletManagerUi.ts

```diff
@@ -15,11 +14,9 @@ import { observableWalletProperties } from '../observableWallet';
  * requires a unique `walletId`, which is calculated by {@link WalletManagerUi} before passing the message to
  * {@link WalletManagerWorker}. It is not meant to be calculated or maintained by the user.
  */
-export class WalletManagerUi implements Shutdown {
+export class WalletManagerUi<Metadata extends {}> implements Shutdown {
```

Comment:
Let's delete this as you suggested in DM. Then we can rename WalletManagerWorker to just WalletManager

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1012/commits/813029a3123dac04ffb0f783c6e239a3c19ae8e9